docs/UserGuide.md

```diff
@@ -402,6 +402,11 @@ Furthermore, certain edits can cause the application to behave in unexpected way
 --------------------------------------------------------------------------------------------------------------------
 <div style="page-break-after: always;"></div>
 
+## Glossary
+* **CLI**: Command-line interface where you interact with the system using your keyboard.
+* **GUI**: is an interface between a user and a computer program or website which uses visual elements in order to interact with the user.
```

Comment:
paiseh, I don't want be nitty gritty
GUI: an interface between a user and a computer program or website that uses visual elements to interact with the user.

Reply:
I have changed the line altogether as per a request above